creator-node/src/fileManager.js

```diff
@@ -12,8 +12,6 @@ const { logger: genericLogger } = require('./logging')
 const { sendResponse, errorResponseBadRequest } = require('./apiHelpers')
 const DecisionTree = require('./utils/decisionTree')
 const { asyncRetry } = require('./utils/asyncRetry')
```

Comment:
can this whole file be ignored bc of bad merge?

Reply:
yeah bad merge. fixed that now